eve/main.yml

```
@@ -1553,3 +1556,300 @@ stages:
+          command: >
+            ssh -F ssh_config bootstrap
+            sudo kubectl --kubeconfig /etc/kubernetes/admin.conf apply -f
+            /tmp/examplesolution_v1alpha1_versionserver_cr.yaml
+          workdir: build/eve/workers/openstack-multiple-nodes/terraform/
+          haltOnFailure: true
+      - ShellCommand:
+          name: Verify example-solution-operator Deployment
+          env:
+            KUBECONFIG: "/etc/kubernetes/admin.conf"
+          command: |
+            ssh -F ssh_config bootstrap --
+            bash -c '
+            ATTEMPTS=0
+            declare -a GET_REPLICAS=(
+                sudo kubectl --kubeconfig $KUBECONFIG get deployments
+                example-solution-operator -n example-environment
+                -o custom-columns=:status.availableReplicas
+            )
+            REPLICAS=$("${GET_REPLICAS[@]}")
+            until [ $REPLICAS -eq 1 ]; do
+              ((ATTEMPTS++))
+              if [ $ATTEMPTS -eq 10 ]; then
+                echo "Could not obtain Replicas after ${ATTEMPTS} attempts" >&2
+                exit 1
+              fi
+              sleep 3
+              REPLICAS=$("${GET_REPLICAS[@]}")
+            done'
+          workdir: build/eve/workers/openstack-multiple-nodes/terraform/
+          haltOnFailure: true
+      - ShellCommand:
+          name: Verify exampleversions Deployment
+          env:
+            KUBECONFIG: "/etc/kubernetes/admin.conf"
+          command: |
+            ssh -F ssh_config bootstrap --
+            bash -c '
+            ATTEMPTS=0
+            declare -a GET_REPLICAS=(
+                sudo kubectl --kubeconfig $KUBECONFIG get deployments
+                example-versionserver -n example-environment
+                -o custom-columns=:status.availableReplicas
+            )
+            REPLICAS=$("${GET_REPLICAS[@]}")
+            until [ $REPLICAS -eq 3 ]; do
+              ((ATTEMPTS++))
+              if [ $ATTEMPTS -eq 10 ]; then
+                echo "Could not obtain Replicas after ${ATTEMPTS} attempts" >&2
+                exit 1
+              fi
+              sleep 3
+              REPLICAS=$("${GET_REPLICAS[@]}")
+            done'
+          workdir: build/eve/workers/openstack-multiple-nodes/terraform/
+          haltOnFailure: true
+      - SetPropertyFromCommand:
+          name: Set Cluster IP as Property for example-versionserver
+          property: versionserver_cluster_ip
+          env:
+            KUBECONFIG: "/etc/kubernetes/admin.conf"
+            SSH_CONFIG: >-
+              eve/workers/openstack-multiple-nodes/terraform/ssh_config
+          command: |
+            ssh -F $SSH_CONFIG bootstrap --
+            bash -c '
+            declare -a GET_IP=(
+              sudo kubectl --kubeconfig "$KUBECONFIG" get service
+              example-versionserver -n example-environment
+              -o custom-columns=:spec.clusterIP
+            )
+            IP=$("${GET_IP[@]}")
+            echo $IP'
```

Why not just execute the command instead of this:
```suggestion
            ssh -F $SSH_CONFIG bootstrap --
            sudo kubectl --kubeconfig "$KUBECONFIG" get service
            example-versionserver -n example-environment
            -o custom-columns=:spec.clusterIP
```

---

Done